### DISCUSSION TITLE
Stripping Down to One Operator

### DISCUSSION THREAD

**Discussion by morgurth** [k8s-operatorhub/community-operators#1240] [Q&A] [answered]
So, I have to work on an airgapped OpenShift install. With the Red Hat Operators, I use a bastion host to do this. I use "opm prune" to specify only those operators I want. It creates a Sqlite catalog. I can then use "oc catalog mirror" to push this into my internal repo. I then create a catalog source pointing to this and use the image content policy mapping. It works great.

However, I need to get just the minio-operator out of Operaorhub.io. The "opm prune" doesn't work. I did pull this down with a "git clone" and deleted all the other directories under operators that I don't need. But, I think I'm a bit confused on how to make a bundle out of it and also add that bundle to a catalog. Can someone point me on the steps to do so?

**Comment by mvalarh** [comment, marked as answer]
Do you want to do it locally for yourself only? If yes you can use `opm index rm` and it will remove `minio-operator` for you.
